src/components/layout/LeftBorderContainer/LeftBorderContainer.stories.tsx

```diff
@@ -0,0 +1,32 @@
+import type { ComponentMeta, ComponentStory } from '@storybook/react';
+import LeftBorderContainer from './LeftBorderContainer';
+
+export default {
+  title: 'Layout/LeftBorderContainer',
+  component: LeftBorderContainer,
+  argTypes: {},
+} as ComponentMeta<typeof LeftBorderContainer>;
+
+const Template: ComponentStory<typeof LeftBorderContainer> = (args) => (
+  <LeftBorderContainer {...args} />
```

Comment:
Let's add some content here so we can see that it's displayed. Also let's make it enabled by default in the story